### DISCUSSION TITLE
ironpython error

### DISCUSSION THREAD

**Discussion by clara12345678910** [bonsai-rx/bonsai#1631] [Q&A]
Hi everybody!
I'm a new user of Bonsai. I have a problem with ironpython because when I insert a python transform node after an operator like zip it appears this message:
"type "ironpython.runtime.types.oldclass" could not be loaded publickeytoken=7f709c5b713576e1"
And i can not start the workflow. could anyone help me please? thank you very much 
(i've downloaded the package iron python standard library and scripting)

**Comment by bruno-f-cruz** [comment]
Add your workflow as requested in #864

**Reply by clara12345678910** [reply]
[speed2.zip](https://github.com/bonsai-rx/bonsai/files/13403729/speed2.zip)
![ironpython](https://github.com/bonsai-rx/bonsai/assets/151395988/7eb69f0b-d1ce-4714-ab50-b30fbfa7e16e)

**Reply by bruno-f-cruz** [reply]
Can you also add your bonsai.config file located next to the executable (bonsai.exe)? Thanks

**Reply by clara12345678910** [reply]
[Bonsaiconfig.txt](https://github.com/bonsai-rx/bonsai/files/13403817/Bonsaiconfig.txt)

I attached it. I converted it to txt cause github doesn't accept .config files. Hope you can open and read it correctly

**Reply by bruno-f-cruz** [reply]
It seems you installed a version of ironpython that is not compatible with the python package. You are trying to run 3.4.1 while the python package only supports up to 2.8.0

**Reply by addy9908** [reply]
I have the same problem when I update the packages, then I can not go back to ironpython 2.75. Any solution?

**Reply by bruno-f-cruz** [reply]
You can keep your environment version controlled so you can always role back in the future. 
https://bonsai-rx.org/docs/articles/environments.html

For now, I think the easiest would be to uninstall ironpython 2.8 via the package manager, and re-install the Bonsai python package again. It should install the correct ironpython version dependency
